test/test_integration_credentials_keyring.py

```diff
@@ -18,13 +18,13 @@
 from test.util.test_runners.integration_test_case import IntegrationTestCaseBase
 from test.util import test_helpers as utils
 
-from conjur.constants import  DEFAULT_CONFIG_FILE, DEFAULT_CERTIFICATE_FILE,DEFAULT_NETRC_FILE
+from conjur.constants import DEFAULT_CONFIG_FILE, DEFAULT_CERTIFICATE_FILE, DEFAULT_NETRC_FILE
 
 
-class CliIntegrationTestCredentials(IntegrationTestCaseBase):
+class CliIntegrationTestCredentialsKeyring(IntegrationTestCaseBase):
```

Comment:
You are missing the equivalent of `def test_no_netrc_and_logout_returns_successful_logout_message(self):` for keyring